### DISCUSSION TITLE
Unrecognized open source license ?

### DISCUSSION THREAD

**Discussion by yiyungent** [community/community#43365] [Repositories]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/16939388/210728691-3644b943-1fd7-4990-8708-68a9c65a1ede.png)

https://github.com/yiyungent/PluginCore

`Apache-2.0` unrecognized, 

The same file (LICENSE) in my other repository can be recognized normally

https://github.com/yiyungent/remember

Why ?

**Comment by 2096779623** [comment]
Try to delete it,then re-run license template generation.
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository
